csharp/src/IceDiscovery/Locator.cs

```diff
@@ -92,14 +92,13 @@ public ValueTask SetAdapterDirectProxyAsync(string adapterId, IObjectPrx? proxy,
         public ValueTask SetServerProcessProxyAsync(string id, IProcessPrx? process, Current current) =>
             new ValueTask(Task.CompletedTask);
 
-        internal IObjectPrx? FindAdapter(string adapterId, out bool isReplicaGroup)
+        internal (IObjectPrx? proxy, bool IsReplicaGroup) FindAdapter(string adapterId)
```

Comment:
This tuple name is inconsistent - should be `Proxy`.

Reply:
Fixed